BuildScript.Nuke/Components/IDependDB.cs

```diff
@@ -0,0 +1,173 @@
+          .DependsOn<IPack>()
+          .Requires(() => DependDBProjectName)
+          .Requires(() => DependDBProjectBranch)
+          .Requires(() => DependDBRetentionTime)
+          .Requires(() => DependDBProjectImportNotificationMailAddress != null)
+          .Requires(() => VcsUrlTemplate)
+          .Requires(() => DependDBTargetFramework)
+          .Requires(() => DependDBTrackedReferences)
+          .Executes(() =>
+          {
+              if (NugetPreProcessor == null)
+                  throw new InvalidOperationException("Cannot find the DependDB Nuget pre processor. Please install the NuGet package 'DependDB.BuildProcessor.NuGetPreProcessor'.");
+              if (BuildProcessor == null)
+                  throw new InvalidOperationException("Cannot find the DependDB build processor. Please install the NuGet package 'DependDB.BuildProcessor'.");
+
+              var outputFolder = OutputFolder / "DependDB";
+              var buildProcessorFile = TempFolder / "DependDB.BuildProcessor.config.xml";
+
+              var projectVersion = DependDBProjectVersion ?? BuildMetadataPerConfiguration["Debug"].AssemblyVersion;
+
+              var trackedReferences = DependDBTrackedReferences.Split(';');
+              var nugetSources = DependDBNuGetSources != null
+                      ? DependDBNuGetSources.Split(';')
+                      : [];
+
+              if (!int.TryParse(DependDBRetentionTime, out var retentionTime))
+                  throw new FormatException($"The specified parameter '{nameof(DependDBRetentionTime)}' is not an integer.");
+
+              var ownPackagesDirectory = DetermineOwnPackagesDirectory();
+              if (!ownPackagesDirectory.GetFiles("*.nupkg").Any())
+                  throw new InvalidOperationException("Own packages directory does not contain any NuGet packages.");
+
+              Log.Information("DependDB configuration:");
+              Log.Information($"  OwnPackagesDirectory: '{ownPackagesDirectory}'");
+              Log.Information($"  {nameof(DependDBProjectName)}: '{DependDBProjectName}'");
+              Log.Information($"  {nameof(DependDBProjectBranch)}: '{DependDBProjectBranch}'");
+              Log.Information($"  {nameof(DependDBProjectVersion)}: '{projectVersion}'");
+              Log.Information($"  {nameof(DependDBRetentionTime)}: '{retentionTime}'");
+              Log.Information($"  {nameof(DependDBProjectImportNotificationMailAddress)}: '{DependDBProjectImportNotificationMailAddress}'");
+              Log.Information($"  {nameof(DependDBProjectImportNotificationMailAddressCc)}: '{DependDBProjectImportNotificationMailAddressCc}'");
+              Log.Information($"  {nameof(VcsUrlTemplate)}: '{VcsUrlTemplate}'");
+              Log.Information($"  {nameof(DependDBTargetFramework)}: '{DependDBTargetFramework}'");
+              Log.Information($"  {nameof(DependDBTrackedReferences)}: [{string.Join(", ", trackedReferences.Select(e => $"'{e}'"))}]");
```

Comment:
The entries should not be too long. It would improve the output a bit but the code here gets a bit more unreadable because of the required indent. The output here is just meant to aid in debugging so I think the current solution is fine.